### PR TITLE
Jest jointjs

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

It's part one so I won't close ticket yet

closes  --- 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: jestSetupAfterEnv.js [hunk 1]
```diff
@@ -6,3 +6,7 @@ import "@testing-library/jest-dom";
  * we increase the default timeout here.
  */
 jest.setTimeout(10000);
+
+//JointJS mock to make library work
+window.SVGAngle = jest.fn();
```
Comment (LauraStordeur) [inline jestSetupAfterEnv.js:11]: why did you need it twice?

File: src/Test/Utils/custom-queries.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import {
+  queryHelpers,
+  buildQueries,
+  Matcher,
+  MatcherOptions,
+} from "@testing-library/react";
+
+const getMultipleError = (attrName: string) => (_c, selectorValue) =>
```
Comment (LauraStordeur) [inline src/Test/Utils/custom-queries.ts:8]: aren't these supposed to be in the "words" file?

Comment (LauraStordeur) [reply src/Test/Utils/custom-queries.ts:8]: The composer is part of the app, the words file is a dictionary of any pieces of text (or at least should, there are places where it wasn't used out of laziness I assume) :D So you can add those two in the file :D

Comment (matborowczyk) [reply src/Test/Utils/custom-queries.ts:8]: I wouldn’t mix error messages from tests with displayed text, but I don't mind, I'll add those :D

**Review by LauraStordeur — APPROVED** [review]
